index.html

```diff
@@ -26,7 +32,158 @@ <h1>I’m Mariane Glad to see you!</h1>
+                <li class="lang"><a href="#">html</a></li>
+                <li class="lang"><a href="#">css</a></li>
+                <li class="lang"><a href="#">javaScript</a></li>
+            </ul>
+            <div>
+                <button class="btn" type="button" class="default_btn">See Project</button>
+            </div>
+        </div>
+
+        <!--Portfolio Card -4-->
+        <div class="portfolio-snapshot">
+            <a class=img href="#"><img src="images/portfolio-snapshoot-4.svg" alt="Portfolio-Snapshot-4"></a>
+
+            <h1 class="project-title">Multi-Post Stories</h1>
+            <ul class="project-name">
+                <li class="sub-title"><a href="#">CANOPY</a></li>
+                <li class="bullet">&#x2022;</li>
+                <li><a class="backend-yr" href="#">Back End Dev</a></li>
+                <li class="bullet">&#x2022;</li>
+                <li><a class="backend-yr" href="#">2015</a></li>
+            </ul>
+            <p class="description">A daily section of privately personalized reads; no accounts or sign-ups
+                required.
+            </p>
+
+            <ul class="classification">
+                <li class="lang"><a href="#">html</a></li>
+                <li class="lang"><a href="#">css</a></li>
+                <li class="lang"><a href="#">javaScript</a></li>
+            </ul>
+            <div>
+                <button class="btn" type="button" class="default_btn">See Project</button>
+            </div>
+        </div>
     </section>
+
+
+    <!-- About section -->
+    <section id="about-section">
+        <div class="about">
+            <h1>About Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.</p>
+            <a href="#">LET’S CONNECT</a>
+            <ul class="social-links-2">
+                <li><a href="#"><img src="images/Github.svg" alt="Github"></a></li>
+                <li><a href="#"><img src="images/Linkedin.svg" alt="LinkedIn"></a></li>
+                <li><a href="#"><img src="images/Angel List.svg" alt="Angel List"></a></li>
+                <li><a href="#"><img src="images/Twitter.svg" alt="Twitter"></a></li>
+                <li><a href="#"><img src="images/Medium.svg" alt="Medium"></a></li>
+            </ul>
+            <div>
+                <button class="btn" type="button" class="default_btn">Get my resume</button>
+            </div>
+
+            <div class="language">
+                <div class="title border-none">
+                <h5>Languages</h5>
+                <img src="images/Dropdown-down arrow.svg" alt="dropdown-arrow">
+            </div>
+                <ul class="classification-2">
+                    <li><a href="#"><img src="images/js-logo.svg" alt="javascript"></a>JavaScript</li>
```

Comment:
- [ ] I think it's better to avoid lines of code longer than 100 characters following the best practices of coding. For more information [click here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Reply:
Thank for the information. Since this is not a project requirement, I will take this as optional. Thank you.